test/test_workspace/WORKSPACE

```diff
@@ -11,37 +11,34 @@ load(
 
 maven_repository_specification(
     name = "maven",
-    insecure_artifacts = [
-        "com.google.dagger:dagger:2.20",
-        "com.google.dagger:dagger-compiler:2.20",
-        "com.google.dagger:dagger-producers:2.20",
-        "com.google.dagger:dagger-spi:2.20",
-        "com.google.code.findbugs:jsr305:3.0.2",
-        "com.google.errorprone:javac-shaded:9+181-r4173-1",
-        "com.google.googlejavaformat:google-java-format:1.6",
-        "com.squareup:javapoet:1.11.1",
-        "org.checkerframework:checker-compat-qual:2.5.5",
-        "javax.annotation:jsr250-api:1.0",
-        "javax.inject:javax.inject:1",
-        "junit:junit:4.13-beta-1",
-        "com.google.errorprone:error_prone_annotations:2.1.3",
-        "com.google.j2objc:j2objc-annotations:1.1",
-        "org.codehaus.mojo:animal-sniffer-annotations:1.14",
-        "org.hamcrest:hamcrest-core:1.3",
-    ],
     artifacts = {
-        # For illustration, most artifacts should actually be in this form, to ensure that only known .jar contents
-        # are imported into the workspace. (repository injection attacks via DNS poisoning are no fun)
-        "com.google.guava:guava:25.0-jre": "3fd4341776428c7e0e5c18a7c10de129475b69ab9d30aeafbb5c277bb6074fa9",
+        # This is the proper way to specify an artifact
+        "com.google.guava:guava:25.0-jre": { "sha256": "3fd4341776428c7e0e5c18a7c10de129475b69ab9d30aeafbb5c277bb6074fa9" },
+        "com.google.dagger:dagger:2.20":   { "sha256": "d37a556d8d57e2428c20e222b95346512d11fcf2174d581489a69a1439b886fb" },
+        # This is how you specify an artifact that has no hash.  You must either supply a sha256 hash of the jar file
```

Comment:
Second sentence isn't

Reply:
Second sentence continues on the next line?